NEWS.md

```diff
@@ -13,15 +15,20 @@
 
 ## Environments
 
-1. Morphotype and Genotype association to environment (Water table, DEM and derivates)
+1. Morphotype and Genotype association to environment (Water table, DEM and derivates)(GbS genotypes)
 1. Proxy of water table map with DEM
 
 ## Genetic
 
+1. Exclude individuals with conflictive gene pool assignment (<-comparison of different genetic marker analyses)
+
 ## Functional
 
 1. Functional variability intra vs inter individuals (BRIDGE)
```

Comment:
Apparently BRIDGE raw data with leaf level informations are no more available, but I'll still give it a try